### PR TITLE
RU_private_data_tutorial

### PR DESCRIPTION
**Pull request by zemtsov**
Signed-off-by: Alexander Zemtsov <a.zemtsov@gmail.com>

### CODE REVIEW

File: docs/locale/ru_RU/source/private_data_tutorial.rst [hunk 1]
```diff
@@ -1,20 +1,13 @@
-Using Private Data in Fabric
-============================
+Использование закрытых данных в сетях Fabric
```
Comment (C0rWin) [inline docs/locale/ru_RU/source/private_data_tutorial.rst]:
```suggestion
Использование приватных данных в сетях Fabric
```

File: docs/locale/ru_RU/source/private_data_tutorial.rst [hunk 2]
```diff
@@ -1,20 +1,13 @@
...
+============================================
 
-This tutorial will demonstrate the use of collections to provide storage
-and retrieval of private data on the blockchain network for authorized peers
-of organizations.
+В этом учебном руководстве демонстрируется использование подгрупп допуска авторизованными одноранговыми узлами организаций для хранения и доступа к закрытым данным в блокчейн-сети.
```
Comment (C0rWin) [inline docs/locale/ru_RU/source/private_data_tutorial.rst]:
```suggestion
В этом учебном руководстве демонстрируется использование приватных коллекций для хранения закрытых данных в блокчейн-сети доступ к которым имеют только авторизованные одноранговые узлы организаций.
```

File: docs/locale/ru_RU/source/private_data_tutorial.rst [hunk 3]
```diff
@@ -1,20 +1,13 @@
...
 
-.. note:: These instructions use the new Fabric chaincode lifecycle introduced
-          in the Fabric v2.0 release. If you would like to use the previous
-          lifecycle model to use private data with chaincode, visit the v1.4
-          version of the `Using Private Data in Fabric tutorial <https://hyperledger-fabric.readthedocs.io/en/release-1.4/private_data_tutorial.html>`__.
+.. note:: В этом руководстве используется новый жизненный цикл чейнкода Fabric, добавленный в версии 2.0. Если для использования закрытых данных с чейнкодом требуется предыдущая модель жизненного цикла, смотрите «Пример использования закрытых данных в сетях Fabric <https://hyperledger-fabric.readthedocs.io/en/release-1.4/private_data_tutorial.html>`__ для версии 1.4.
 
-The tutorial will take you through the following steps to practice defining,
-configuring and using private data with Fabric:
+Это руководство состоит из следующих подразделов, которые помогут разобраться с определением, настройкой и использованием закрытых данных в сетях Fabric:
```
Comment (C0rWin) [inline docs/locale/ru_RU/source/private_data_tutorial.rst]:
```suggestion
Это руководство состоит из следующих подразделов, которые помогут разобраться с определением, настройкой и использованием приватных данных в сетях Fabric:
```

File: docs/locale/ru_RU/source/private_data_tutorial.rst [hunk 4]
```diff
@@ -26,61 +19,36 @@ configuring and using private data with Fabric:
...
-This tutorial will deploy the `marbles private data sample <https://github.com/hyperledger/fabric-samples/tree/{BRANCH}/chaincode/marbles02_private>`__
-to the Fabric test network to demonstrate how to create, deploy, and use a collection of
-private data. You should have completed the task :doc:`install`.
+В руководстве описывается развертывание `примера закрытых данных marbles <https://github.com/hyperledger/fabric-samples/tree/{BRANCH}/chaincode/marbles02_private>`__ в примере сети Fabric, чтобы продемонстрировать, как создавать, развертывать и использовать подгруппы допуска к закрытым данным. Перед тем как приступить к изучению, следует выполнить шаги, описанные в разделе :doc:`install`.
 
 .. _pd-build-json:
 
-Build a collection definition JSON file
----------------------------------------
+Создание файла JSON определения подгруппы допуска
```
Comment (C0rWin) [inline docs/locale/ru_RU/source/private_data_tutorial.rst]:
```suggestion
Создание файла JSON определения допуска
```

File: docs/locale/ru_RU/source/private_data_tutorial.rst [hunk 5]
```diff
@@ -26,61 +19,36 @@ configuring and using private data with Fabric:
...
-The first step in privatizing data on a channel is to build a collection
-definition which defines access to the private data.
+Чтобы обеспечить конфиденциальность данных в канале, следует сперва создать определение подгруппы допуска, в котором описывается правила доступа к закрытым данным.
 
-The collection definition describes who can persist data, how many peers the
-data is distributed to, how many peers are required to disseminate the private
-data, and how long the private data is persisted in the private database. Later,
-we will demonstrate how chaincode APIs ``PutPrivateData`` and ``GetPrivateData``
-are used to map the collection to the private data being secured.
+В определении подгруппы допуска указывается, кто может сохранять данные, на скольких одноранговых узлах данные будут распределены, сколько одноранговых узлов требуется для распространения закрытых данных, а также время существования закрытых данных в соответствующей базе данных. Далее будет показано, каким образом можно использовать функции API-интерфейса чейнкода ``PutPrivateData`` и ``GetPrivateData`` для связывания подгруппы допуска с закрытыми данными.
```
Comment (C0rWin) [inline docs/locale/ru_RU/source/private_data_tutorial.rst]:
```suggestion
В определении подгруппы допуска указывается, кто может сохранять данные, на скольких одноранговых узлах данные будут распределены, сколько одноранговых узлов требуется для распространения закрытых данных, а также время существования приватных данных в соответствующей базе данных. Далее будет показано, каким образом можно использовать функции API-интерфейса чейнкода ``PutPrivateData`` и ``GetPrivateData`` для связывания подгруппы допуска с закрытыми данными.
```

File: docs/locale/ru_RU/source/private_data_tutorial.rst [hunk 6]
```diff
@@ -26,61 +19,36 @@ configuring and using private data with Fabric:
...
 
-To illustrate usage of private data, the marbles private data example contains
-two private data collection definitions: ``collectionMarbles``
-and ``collectionMarblePrivateDetails``. The ``policy`` property in the
-``collectionMarbles`` definition allows all members of  the channel (Org1 and
-Org2) to have the private data in a private database. The
-``collectionMarblesPrivateDetails`` collection allows only members of Org1 to
-have the private data in their private database.
+Использование закрытых данных далее демонстрируется на примере закрытых данных marble, содержащем два определения подгрупп допуска к закрытым данным: ``collectionMarbles`` и ``collectionMarblePrivateDetails``. Параметр ``policy`` определения подгруппы допуска ``collectionMarbles`` позволяет всем членам канала (организациям Org1 и Org2) хранить закрытые данные в закрытой базе данных. Подгруппа допуска ``collectionMarblesPrivateDetails`` позволяет только членам организации Org1 хранить закрытые данные в собственной закрытой базе данных.
```
Comment (C0rWin) [inline docs/locale/ru_RU/source/private_data_tutorial.rst]:
```suggestion
Использование закрытых данных далее демонстрируется на примере приватных данных marble, содержащем два определения подгрупп допуска к закрытым данным: ``collectionMarbles`` и ``collectionMarblePrivateDetails``. Параметр ``policy`` определения подгруппы допуска ``collectionMarbles`` позволяет всем членам канала (организациям Org1 и Org2) хранить закрытые данные в закрытой базе данных. Подгруппа допуска ``collectionMarblesPrivateDetails`` позволяет только членам организации Org1 хранить закрытые данные в собственной закрытой базе данных.
```

File: docs/locale/ru_RU/source/private_data_tutorial.rst [hunk 7]
```diff
@@ -106,26 +74,20 @@ topic.
...
-The next step in understanding how to privatize data on a channel is to build
-the data definition in the chaincode. The marbles private data sample divides
-the private data into two separate data definitions according to how the data will
-be accessed.
+Следующий шаг обеспечения конфиденциальности данных в канале — это создание определения данных в чейнкода. В примере marbles закрытые данные разделены на два отдельных определения данных для ограничения доступа разных организаций.
 
 .. code-block:: GO
 
- // Peers in Org1 and Org2 will have this private data in a side database
+ // Одноранговые узлы организаций Org1 и Org2 будут хранить эти закрытые данные в базе данных вне сети
```
Comment (C0rWin) [inline docs/locale/ru_RU/source/private_data_tutorial.rst]:
```suggestion
 // Одноранговые узлы организаций Org1 и Org2 будут хранить эти приватные данные в базе данных вне сети
```

File: docs/locale/ru_RU/source/private_data_tutorial.rst [hunk 8]
```diff
@@ -134,66 +96,48 @@ be accessed.
    Owner      string `json:"owner"`
  }
 
- // Only peers in Org1 will have this private data in a side database
+ // Только одноранговые узлы организации Org1 будут хранить эти закрытые данные в базе данных вне сети
```
Comment (C0rWin) [inline docs/locale/ru_RU/source/private_data_tutorial.rst]:
```suggestion
 // Только одноранговые узлы организации Org1 будут хранить эти приватные данные в базе данных вне сети
```

**Comment by ryjones** [comment]
@zemtsov could you compress this to a single commit?

**Review by C0rWin — APPROVED** [review]

**Pull request merged**